pkg/pool-utils/test/RecoveryModePool.test.ts

```diff
@@ -0,0 +1,347 @@
+import { ethers } from 'hardhat';
+import { expect } from 'chai';
+import { Contract } from 'ethers';
+import { SignerWithAddress } from '@nomiclabs/hardhat-ethers/dist/src/signer-with-address';
+import * as expectEvent from '@balancer-labs/v2-helpers/src/test/expectEvent';
+import { deploy } from '@balancer-labs/v2-helpers/src/contract';
+import { Account } from '@balancer-labs/v2-helpers/src/models/types/types';
+import { actionId } from '@balancer-labs/v2-helpers/src/models/misc/actions';
+import { BigNumberish, fp } from '@balancer-labs/v2-helpers/src/numbers';
+import { PoolSpecialization } from '@balancer-labs/balancer-js';
+import { MONTH } from '@balancer-labs/v2-helpers/src/time';
+import { ANY_ADDRESS, ZERO_ADDRESS } from '@balancer-labs/v2-helpers/src/constants';
+
+import TokenList from '@balancer-labs/v2-helpers/src/models/tokens/TokenList';
+import TypesConverter from '@balancer-labs/v2-helpers/src/models/types/TypesConverter';
+
+describe('RecoveryModePool', function () {
```

Comment:
All of these are duplicates, is there a reason to add them or just a merge artifact?